Cargo.toml

```diff
@@ -19,9 +19,9 @@ include = ["src/", "LICENSE-*", "README.md", "CHANGELOG.md", "COPYRIGHT"]
 
 [package.metadata.docs.rs]
 # To build locally:
-# RUSTDOCFLAGS="--cfg doc_cfg -Zunstable-options --generate-link-to-definition" cargo +nightly doc --all --all-features --no-deps --open
+# RUSTDOCFLAGS="--cfg docsrs -Zunstable-options --generate-link-to-definition" cargo +nightly doc --all --all-features --no-deps --open
 all-features = true
-rustdoc-args = ["--cfg", "doc_cfg", "-Zunstable-options", "--generate-link-to-definition"]
+rustdoc-args = ["--cfg", "docsrs", "-Zunstable-options", "--generate-link-to-definition"]
```

Comment:
Do we steel need `-Zunstable-options`?

Reply:
IIRC it was there for `--generate-link-to-definition`

Reply:
We then should add it to `rand_core` and `rand_distr` as well. Although, I am not sure if this feature even works right now. The link in #1327 does not contain any links to definitions anymore.